### ISSUE TITLE
Show SNV/indels in SMN genes via button redirect from the SMN CN page

### ISSUE THREAD

**Issue by Jakob37** [Clinical-Genomics/scout#4822]
**Is your feature request related to a problem in the current program to new available techology or software? Please describe and add links/citations if appropriate.**

Our local geneticists are looking into using the SMN CN module in their clinical routine, and raised some points for discussion. I'll open issues for their discussion points to discuss.

**Describe the solution you'd like**

They asked whether it would be possible to add a panel showing what SNV/indels are residing inside the SMN genes similarly to how it is on the SV panel page.

I.e. this panel:

![overlapping_snvs](https://github.com/user-attachments/assets/50b3cb0a-c4a6-499d-8c4b-23d033bf7364)

**Comment by dnil** [comment]
Right, this _could_ be done with the current setup. The most salient reason not to is that at least our short read pipelines are not guaranteed to give (and with that I mean essentially guaranteed to not give 😉) very reliable results on pseudogene/genefamily genes. The calls on the SMN page are instead reasonably reliable, up to the performance reported for the SMNCopyNumberCaller. The developer felt the CLGs interpreting might not appreciate the difference. 😊

**Comment by Jakob37** [comment]
OK I see, that makes sense! I'll forward that thought, let's see what they say.

**Comment by Jakob37** [comment]
I asked our geneticists about this. They decided to check in our data for themselves whether this would be of value, and screened variants from our ~8000 wgs cases for this region.

They came back with some points:

* Sequencing of exon 8 in SMN1 has nice coverage and unique mapping due to having enough sequence differences between the two genes.
* They only saw a few artefact calls in SMN1 with relevant rank scores - most calls here were in introns or with negative rank score. For the few variants scored high enough for consideration, they say they can easily check for unique mapping in IGV.
* They also saw a handful (3) real variants with rank score > 0 including one in exon 8.
* They see in ClinVar that there is a bunch of pathogenic variants in SMN1, out of which some are in exon 8.
* They want to assess variants in exon 8 in the cases where cases have just one copy number of SMN1 plus a variant in the other gene (which causes 5% of SMA cases according to them).

In short, in their method document they want to have a safety-check for one copy SMN1 plus exon 8 SNV/indel in the other copy. Having this available in Scout together with where they check the SMN calls would make that check much easier to do.

Sounds to me that they are making some good points 🤔 What do you say @dnil ?

**Comment by dnil** [comment]
Yes, there are a few parts of the gene that are decent, while not large, it is of course unnecessary to miss something. Sounds like they would go search for them anyway - it’s not like we are hiding the calls on the SNV view We could always add a warning to the variants table - or to save a lot of code just add a button to the snv and sv view prefilled with a SMN1 & SMN2 query…

**Comment by ehre** [comment]
My two cents on this - with current srWGS the SMNCopyNumberCaller is sufficient for screening purposes. I would rather that we put our energy into a high quality SMN1 (and SMN2) calling, phasing and visualization for soon to come lrWGS data.

**Comment by Jakob37** [comment]
OK I see! I discussed it briefly with the other geneticists here as well. In conclusion it seems like if adding a button with a prefilled search would be fully sufficient for what they want to achieve, and would be easy to add. So maybe that is the way to go then 👍

**Comment by dnil** [comment]
If you feel for this one @Jakob37, perphas look at the SNV/SV buttons on the WTS page, and make the search for SMN1 and SMN2 instead of the gene with an RNA outlier!

**Comment by Jakob37** [comment]
> If you feel for this one @Jakob37, perphas look at the SNV/SV buttons on the WTS page, and make the search for SMN1 and SMN2 instead of the gene with an RNA outlier!

Sure, I'll look into it!
